plugins/chat/assets/javascripts/discourse/initializers/chat-audio.js

```diff
@@ -15,6 +16,42 @@ export default {
       return;
     }
 
+    this.supportsServiceWorker = () => {
```

Comment:
I'm wondering if this will be useful elsewhere, maybe worth extracting to the capabilities service?